Dockerfile.rocm

```
@@ -27,36 +43,27 @@ ARG BUILD_FA="1"
 ARG BUILD_TRITON="1"
 
 # Install some basic utilities
-RUN apt-get update && apt-get install python3 python3-pip -y
+RUN yum -y install python3 python3-pip python3-devel patch
 
 # Install some basic utilities
-RUN apt-get update && apt-get install -y \
-    curl \
-    ca-certificates \
-    sudo \
-    git \
-    bzip2 \
-    libx11-6 \
-    build-essential \
-    wget \
-    unzip \
-    nvidia-cuda-toolkit \
-    tmux \
- && rm -rf /var/lib/apt/lists/*
+RUN yum -y install git bzip2 libX11-devel wget unzip https://mirror.stream.centos.org/9-stream/BaseOS/x86_64/os/Packages/tmux-3.2a-4.el9.x86_64.rpm && yum clean all
```

Do we really need `tmux`? I'd remove it. Possibly even `libX11-devel`

```suggestion
RUN yum -y install git bzip2 wget unzip
```